frontend/lib/l10n/app_en.arb

```diff
@@ -1,133 +1,129 @@
 {
-  "about_dependencies": "Software-Bibliotheken",
-  "about_developmentOptions": "Entwickleroptionen",
-  "about_disclaimer": "Haftung, Haftungsausschluss und Impressum",
-  "about_license": "Lizenz",
-  "about_licenses": "Lizenzen",
-  "about_moreInformation": "Mehr Informationen",
-  "about_numberLicenses": "{count, plural, =0{Keine Lizenzen} =1{1 Lizenz} other{{count} Lizenzen}}",
-  "about_privacyDeclaration": "Datenschutzerklärung",
-  "about_publisher": "Herausgeber",
-  "about_sourceCode": "Quellcode der App",
-  "about_title": "Über",
+  "about_dependencies": "Libraries",
+  "about_developmentOptions": "Developer options",
+  "about_disclaimer": "Liability, disclaimer and imprint",
+  "about_license": "License",
+  "about_licenses": "Licenses",
+  "about_moreInformation": "More information",
+  "about_numberLicenses": "{count, plural, =0{No licenses} =1{1 License} other{{count} Licenses}}",
+  "about_privacyDeclaration": "Privacy policy",
+  "about_publisher": "Publisher",
+  "about_sourceCode": "Source code",
+  "about_title": "About",
 
-  "category_clothing": "Kleidung",
-  "category_clothingLong": "Kleidung/Gebrauchtes",
-  "category_culture": "Kultur",
-  "category_cultureLong": "Bildung/Kultur/Unterhaltung",
-  "category_cultureLongAlternative": "Kultur/Museen/Freizeit",
-  "category_digitalParticipation": "Digitale Teilhabe",
-  "category_education": "Bildung",
-  "category_fashion": "Mode",
-  "category_fashionLong": "Mode/Beauty",
-  "category_food": "Gastronomie",
-  "category_foodLong": "Essen/Trinken/Gastronomie",
-  "category_health": "Gesundheit",
-  "category_healthLong": "Gesundheit/Sport/Wellness",
-  "category_leisure": "Freizeit",
-  "category_leisureLong": "Freizeit/Reise/Unterkünfte",
-  "category_living": "Einrichtung",
-  "category_livingLong": "Wohnen/Haus/Garten",
-  "category_lunchTables": "Mittagstische",
+  "category_clothing": "Clothing",
+  "category_clothingLong": "Clothing/Second hand",
+  "category_culture": "Culture",
+  "category_cultureLong": "Education/Culture/Entertainment",
+  "category_cultureLongAlternative": "Culture/Museums/Leisure",
+  "category_digitalParticipation": "Digital participation",
+  "category_education": "Education",
+  "category_fashion": "Fashion",
+  "category_fashionLong": "Fashion/Beauty",
+  "category_food": "Restaurants",
```

Comment:
Gastronomie would rather be gastronomy - if it only includes restaurants, I would change the German term to Restaurants